app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -72,11 +72,25 @@ def healthcheck
     end
 
     def facilities
-      render(json: lighthouse_facilities_service.get_facilities(lighthouse_facilities_params))
+      lighthouse_facilities = lighthouse_facilities_service.get_facilities(lighthouse_facilities_params)
+
+      if params[:ves_active]
+        active_facilities = ves_active_facilities(lighthouse_facilities)
+
+        render(json: active_facilities)
+      else
+        render(json: lighthouse_facilities)
+      end
     end
 
     private
 
+    def ves_active_facilities(lighthouse_facilities)
```

Comment:
Would this be better as a method on the model such as `active` or something? I like keeping controllers light. It's not a big deal either way though.